frontend/catalyst/pennylane_extensions.py

```diff
@@ -1389,6 +1389,11 @@ def __call__(self, *args):
 def for_loop(lower_bound, upper_bound, step):
     """A :func:`~.qjit` compatible for-loop decorator for PennyLane/Catalyst.
 
+    .. note::
+
+        Catalyst can now automatically convert Python for loop statements for you. Requires setting
```

Comment:
```suggestion
        Catalyst can automatically convert Python for loop statements for you. Requires setting
```

to make the documentation a bit more 'time invariant' :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/258/commits/063e547880be0260727b13b49cbbb8a5e30a3c38